packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1036,6 +1045,53 @@ export class Project extends ProjectBase {
     };
   }
 
+  private renderLogs(props: ProjectProps): CfnProject.LogsConfigProperty {
+    let s3Config: CfnProject.S3LogsConfigProperty|undefined;
+    let cloudwatchConfig: CfnProject.CloudWatchLogsConfigProperty|undefined;
+
+    if (props.logging?.s3) {
+      const s3Logs = props.logging.s3;
+
+      let status = 'ENABLED';
+      if ('enabled' in s3Logs) {
+        status = s3Logs.enabled ? 'ENABLED' : 'DISABLED';
+      }
+
+      s3Config = {
+        status,
+        location: `${s3Logs.bucket.bucketName}${s3Logs.prefix}`,
+        encryptionDisabled: s3Logs.encrypted ?? false,
+      };
+    } else {
+      s3Config = undefined;
+    }
+
+    if (props.logging?.cloudwatch) {
+      const cloudWatchLogs = props.logging.cloudwatch;
+      let status = 'ENABLED';
+      if ('enabled' in cloudWatchLogs) {
+        status = cloudWatchLogs.enabled ? 'ENABLED' : 'DISABLED';
+      }
+
+      if (status === 'ENABLED' && !(cloudWatchLogs.logGroup)) {
+        throw new Error('A log group is required if cloudwatch logging is enabled');
+      }
+
+      cloudwatchConfig = {
+        status,
+        groupName: cloudWatchLogs.logGroup?.logGroupName,
+        streamName: cloudWatchLogs.prefix,
+      };
+    } else {
+      cloudwatchConfig = undefined;
+    }
```

Comment:
Same comments here as above (no need for the `'enabled` in cloudWatchLogs` expression, the `else` part).